monitoring/prometheus/alertmanager_staging.yml

```diff
@@ -0,0 +1,37 @@
+global:
+  resolve_timeout: 10s
+  slack_api_url: 'https://hooks.slack.com/services/x/x/x'
```

Comment:
No need to set up a separate alertmanager config if both of them will have slack webhook url, channel set to dummy value. We can use a single `.yml` file and push it to s3 with populated values